SPECS/perl-Authen-SASL/perl-Authen-SASL.spec

```diff
@@ -2,7 +2,7 @@ Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Name:           perl-Authen-SASL
 Version:        2.16
-Release:        21%{?dist}
+Release:        22%{?dist}
```

Comment:
Please lint the spec and perform a thorough spec clean-up (see: [instructions](https://dev.azure.com/mariner-org/mariner/_wiki/wikis/mariner.wiki/125/Introduction-to-SPEC-Files)). The core specs are required to follow more strict guidelines, so there's usually some additional work required when we migrate extended specs.

Reply:
Sounds good, thank you for pointing this out :)

I will run the spec-cleaner on both the perl-Authen-SASL.spec and perl-GSSAPI.spec and verify that the tests pass